device-types/Digi/Digi-CM-32.yaml

```diff
@@ -5,6 +5,10 @@ slug: digi-cm-32
 part_number: '70001908'
 u_height: 1
 is_full_depth: false
```

Comment:
1. Please add the `weight` and associated `weight_unit` data point if you have it or can find it. This appears to be listed in the datasheet (5.80 lbs or 2.60 kg)